pom.xml

```diff
@@ -48,7 +48,6 @@
         <module>inlong-sort-standalone</module>
         <module>inlong-manager</module>
         <module>inlong-dashboard</module>
-        <module>inlong-distribution</module>
```

Comment:
Please check out the GitHub action, we have an action to upload the package of distribution.

Reply:
See https://github.com/apache/inlong/actions/runs/5266513756/workflow?#L91-#L110